src/main/resources/liveTemplates/ATG.xml

```diff
@@ -0,0 +1,15 @@
+<templateSet group="ATG">
+    <template name="trytd" value="try (AutoCloseTransactionDemarcation td =&#10;             new AutoCloseTransactionDemarcation($SELECTION$$TRANSACTION_MANAGER$$END$, TransactionDemarcation.REQUIRED)) {&#10;    // TODO: your code goes here&#10;    td.setSuccess(true);&#10;} catch (TransactionDemarcationException e) {&#10;    throw e;&#10;}" description="Auto-closed transaction template" toReformat="true" toShortenFQNames="true">
+        <variable name="TRANSACTION_MANAGER" expression="" defaultValue="&quot;transactionManager&quot;" alwaysStopAt="true" />
+        <context>
+            <option name="JAVA_CODE" value="true" />
+        </context>
+    </template>
+    <template name="trypm" value="// TODO: Move to constants block (consider using shortcut CTRL+ALT+C)&#10;final String perfMonitorName = &quot;$CURRENT_CLASS_NAME$&quot;;&#10;final String perfMonitorParam = &quot;$CURRENT_METHOD_NAME$&quot;;&#10;try {&#10;    PerformanceMonitor.startOperation(perfMonitorName, perfMonitorParam);&#10;    // TODO: Your code goes here&#10;} finally {&#10;    PerformanceMonitor.endOperation(perfMonitorName, perfMonitorParam);&#10;}&#10;        " description="Perfomance monitor transaction template" toReformat="true" toShortenFQNames="true">
+        <variable name="CURRENT_CLASS_NAME" expression="className()" defaultValue="" alwaysStopAt="true" />
```

Comment:
rename perfMonitorName as invocationClass/monitoredClass.. or some like that
perfMonitorParam - it's closer to invocationMethod/monitoredMethod meaning